java/ql/src/experimental/CWE-939/IncorrectURLVerification.ql

```diff
@@ -0,0 +1,64 @@
+/**
+ * @id java/incorrect-url-verification
+ * @name Insertion of sensitive information into log files
```

Comment:
That name should be updated, it refers to CWE-532.

Reply:
Thanks intrigus-lgtm (@intrigus) for reviewing this PR. Sorry about it and I've removed commits for CWE-532.

Regards,
@luchua-bc

Reply:
Sorry I've corrected the name tag and committed the change.